src/formatDate.js

```diff
@@ -50,7 +50,66 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const DAY_MARK = 'DD';
+  const MONTH_MARK = 'MM';
+
+  const givenDate = date.split(separator);
+  let day;
+  let month;
+  let year;
+
+  let oldYearFormat;
```

Comment:
Maybe you can add some default value just to clarify what these variables are stand for
```suggestion
let day = '';
let month = '';
let year = '';
let oldYearFormat = '';
```